sshcode.go

```diff
@@ -608,3 +629,32 @@ func parseGCPSSHCmd(instance string) (ip, sshFlags string, err error) {
 
 	return strings.TrimSpace(userIP), sshFlags, nil
 }
+
+// gitbashWindowsDir strips a the msys2 install directory from the beginning of
+// the path. On msys2, if a user provides `/workspace` sshcode will receive
+// `C:/msys64/workspace` which won't work on the remote host.
+func gitbashWindowsDir(dir string) string {
+
+	// Don't bother figuring out path if it's relative to home dir.
+	if strings.HasPrefix(dir, "~/") {
+		if dir == "~" {
+			return "~/"
+		}
+		return dir
+	}
+
+	mingwPrefix, err := exec.Command("sh", "-c", "{ cd / && pwd -W; }").Output()
```

Comment:
Does this need to be inside `{}`? I don't think you elaborated on that.

Reply:
I am unsure myself, currently at my linux machine at the moment as my windows got a virus... So i cant test at the moment

Reply:
just did some testing, works either way, honestly i dont think it really matters so i dont see a reason in removing it